meta.go

```diff
@@ -433,6 +472,10 @@ func parseMeta(m map[string]interface{}) (*Meta, error) {
 	}
 	meta.Series = parseStringList(m["series"])
 	meta.Storage = parseStorage(m["storage"])
+	meta.Device, err = parseDevice(m["device"])
```

Comment:
Let's in-line the error check and also do the same for the extra bindings above as a drive by fix